q2_diversity/_beta/_method.py

```diff
@@ -110,19 +113,34 @@ def beta_phylogenetic_alt(table: BIOMV210Format, phylogeny: NewickFormat,
              variance_adjusted=variance_adjusted, bypass_tips=bypass_tips)
 
 
-def beta(table: biom.Table, metric: str, n_jobs: int=1)-> skbio.DistanceMatrix:
-    if metric not in non_phylogenetic_metrics():
+def aitchison(x, y, **kwds):
+    return euclidean(clr(x), clr(y))
+
+
+def beta(table: biom.Table, metric: str,
+         pseudocount: int=1, n_jobs: int=1)-> skbio.DistanceMatrix:
+
+    if not (metric in non_phylogenetic_metrics()):
         raise ValueError("Unknown metric: %s" % metric)
+
+    if metric == 'aitchison':
+        counts = table.matrix_data.toarray().astype(int).T + pseudocount
+        metric = aitchison
+    else:
+        counts = table.matrix_data.toarray().astype(int).T
+
+    validate = True
```

Comment:
Maybe this should just be [inlined](https://github.com/qiime2/q2-diversity/pull/208/files#diff-adf90299393c8099efa932cfcca88186R143), below?